### PR TITLE
T6655 - Add a documentation for the Android SDK.

### PR DESCRIPTION
**Pull request by soohan-ahn**
Created a documentation for the Android SDK, referenced by a documentation of VAN SDK.

### CODE REVIEW

File: android-sdk/README.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+mMolocoView.setAdUnitId(adUnitId);
+mMolocoView.loadAd();
+mMolocoView.setBannerAdListener(this);
+```
+
+Now you are ready use **Moloco Android** for Android devices! 
+
+Once you complete integration, please work with your Moloco contact to verify receipt of the session and other events from Android server.
+
+If there is any question regarding with Moloco Android Android SDK integration, please contact [Moloco](mailto:support@molocoads.com).
```
Comment (myunggeun-moloco) [inline android-sdk/README.md]: nit: Moloco Android Android SDK -> Moloco Android SDK

File: android-sdk/README.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+Initialization requires a `AdUnit ID` provided by Moloco. Please contact [Moloco](mailto:support@molocoads.com) if you need an `AdUnit ID`.
+
+```
+String adUnitId = "your_ad_unit_id";
+mMolocoView.setAdUnitId(adUnitId);
+mMolocoView.loadAd();
+mMolocoView.setBannerAdListener(this);
+```
+
+Now you are ready use **Moloco Android** for Android devices!
```
Comment (myunggeun-moloco) [inline android-sdk/README.md]: nit: maybe Moloco Android -> Moloco Android SDK?

File: android-sdk/README.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    /*
+        Your logic comes here
+    */
+}
+```
+
+### Initialize MolocoView
+
+```
+MolocoView mMolocoView = findViewById(/* Id of your view. ex) R.id.banner_molocoview*/ );
```
Comment (myunggeun-moloco) [inline android-sdk/README.md]: nit: space format at `R.id.banner_molocoview*/ );`

**Review by myunggeun-moloco — COMMENTED** [review]
Great! I left a few minor nits.

**Comment by soohan-ahn** [comment]
@myunggeun-moloco 
Thank you for your check!
I modified on 2231da2.

File: android-sdk/README.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+```
+String adUnitId = "your_ad_unit_id";
+mMolocoView.setAdUnitId(adUnitId);
+mMolocoView.loadAd();
+mMolocoView.setBannerAdListener(this);
+```
+
+Now you are ready use **Moloco Android SDK** for Android devices!
+
+Once you complete integration, please work with your Moloco contact to verify receipt of the session and other events from Android server.
```
Comment (hyeonseo) [inline android-sdk/README.md]: This needs to be rewritten

File: android-sdk/README.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+Initialize a SdkConfiguration with a LogLevel (INFO or DEBUG).
+
+```
+LogLevel logLevel = INFO; // or DEBUG
+final SdkConfiguration sdkConfiguration = new SdkConfiguration(logLevel);
+Moloco.initializeSdk(this, sdkConfiguration);
+```
+
+###  Load an Ad with an ad unit id.
+Initialization requires a `AdUnit ID` provided by Moloco. Please contact [Moloco](mailto:support@molocoads.com) if you need an `AdUnit ID`.
```
Comment (hyeonseo) [inline android-sdk/README.md]: Either `AdUnitID` or `Ad Unit ID`?

File: android-sdk/README.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+
+```java
+apply plugin: 'com.android.application'
+
+android {
+   compileSdkVersion 28
+   buildToolsVersion "28.0.0"
+
+   defaultConfig {
+       applicationId "com.moloco.molocosampleApp"
```
Comment (hyeonseo) [inline android-sdk/README.md]: This looks different from the above screen

**Comment by hyeonseo** [comment]
I left a few comments but they don't appear in Conversation. Do you see them?

**Comment by soohan-ahn** [comment]
I can see 3 comments.
Are there any more comments need to be shown?

**Comment by hyeonseo** [comment]
> I can see 3 comments.
> Are there any more comments need to be shown?

Oh I see them as well. They are all

File: android-sdk/README.md (hunk 4 above)
Comment (soohan-ahn) [reply android-sdk/README.md]: Are there any problems if I remove this line on the first version of document?

Comment (hyeonseo) [reply android-sdk/README.md]: Sounds good to me

File: android-sdk/README.md [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+Initialization requires a `AdUnit ID` provided by Moloco. Please contact [Moloco](mailto:support@molocoads.com) if you need an `AdUnit ID`.
+
+```
+String adUnitId = "your_ad_unit_id";
+mMolocoView.setAdUnitId(adUnitId);
+mMolocoView.loadAd();
+mMolocoView.setBannerAdListener(this);
+```
+
+Now you are ready use **Moloco Android SDK** for Android devices!
```
Comment (linjus) [inline android-sdk/README.md]: 'to' use?

File: android-sdk/README.md [hunk 8]
```diff
@@ -0,0 +1,168 @@
+
+# Moloco Android SDK - Android
+Moloco Android SDK provides ad serving for mobile applications to facilitate advertising for app publishers.
```
Comment (linjus) [inline android-sdk/README.md:3]: Here and below, I'm not sure but I guess `Moloco Android SDK` should not be a naked noun, even though it looks like a proper noun. If it's possible, we could ask a native speaker to proof-read this.

File: android-sdk/README.md [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+mMolocoView.setAdUnitId(adUnitId);
+mMolocoView.loadAd();
+mMolocoView.setBannerAdListener(this);
+```
+
+Now you are ready use **Moloco Android SDK** for Android devices!
+
+Once you complete integration, please work with your Moloco contact to verify receipt of the session and other events from Android server.
+
+If there is any question regarding with Moloco Android SDK integration, please contact [Moloco](mailto:support@molocoads.com).
```
Comment (linjus) [inline android-sdk/README.md]: /regarding with/regarding

File: android-sdk/README.md (hunk 3 above)
Comment (kevin-moloco) [reply android-sdk/README.md]: ex -> e.g.,
ex) R.id.banner_molocoview --> e.g., R.id.banner_molocoview

File: android-sdk/README.md [hunk 10]
```diff
@@ -0,0 +1,168 @@
...
+
+
+## Prerequisites
+Moloco Android SDK 0.0.1 is compatible for devices running Android **API 9** and above.
+
+## Installation
+
+This section contains instructions for installing Android SDK using Android Studio. Moloco Android SDK is built with Android Studio 3.4 Build #AI-183.5429.30.34.5452501. If you’re using a different version and have a problem using Android SDK, please contact [Moloco](mailto:support@molocoads.com).
+
+First, launch your Android Studio, and click `Start a new Android project`. We will name it as `MolocoSampleApp`. On the `Target Android Devices` screen you can click **Next** to choose the defaults. On the next screen, let’s choose `Empty Activity` for this tutorial.
```
Comment (kevin-moloco) [inline android-sdk/README.md]: to choose the defaults -> ... default

File: android-sdk/README.md [hunk 11]
```diff
@@ -0,0 +1,168 @@
...
+
+### Initialize MolocoView
+
+```
+MolocoView mMolocoView = findViewById(/* Id of your view. ex) R.id.banner_molocoview*/ );
+mMolocoView.setWidthPixel(720);
+mMolocoView.setHeightPixel(160);
+```
+
+### SdkConfiguration
```
Comment (kevin-moloco) [inline android-sdk/README.md:145]: Does 'SdkConfiguration' have to be explained ahead of 'Initialize MolocoView'?

File: android-sdk/README.md (hunk 4 above)
Comment (kevin-moloco) [reply android-sdk/README.md]: Android server? Does it have to be Moloco Server? Ah, it will be removed.

**Review by kevin-moloco — DISMISSED** [review]

File: android-sdk/README.md [hunk 12]
```diff
@@ -0,0 +1,168 @@
+
+# Moloco Android SDK - Android
```
Comment (kevin-moloco) [inline android-sdk/README.md]: How about 'Moloco Android SDK' or 'Moloco SDK - Android'?

File: android-sdk/README.md [hunk 13]
```diff
@@ -0,0 +1,168 @@
...
+Moloco Android SDK provides ad serving for mobile applications to facilitate advertising for app publishers.
+
+
+## Prerequisites
+Moloco Android SDK 0.0.1 is compatible for devices running Android **API 9** and above.
```
Comment (kevin-moloco) [inline android-sdk/README.md]: compatible for -> compatible with
Moloco Android SDK 0.0.1 is compatible with all Android devices running Android **API 14** and above.

File: android-sdk/README.md (hunk 11 above)
Comment (soohan-ahn) [reply android-sdk/README.md:145]: I think order of `SdkConfiguration` and `Initialize MolocoView` does not matter actually. These two configurations are independent and does not affect each other.

File: android-sdk/README.md (hunk 8 above)
Comment (soohan-ahn) [reply android-sdk/README.md:3]: I brought this naming by referencing MoPub Android SDK( https://github.com/mopub/mopub-android-sdk ).
Let me know if you have any other idea.

**Comment by soohan-ahn** [comment]
@hyeonseo @kevin-moloco @linjus 
Thank you for your review!
I modified some errors on ce6f5a5 and a241343 .

In addition, I left comments on link below. Let me know if there are more problems concerning these issues.

https://github.com/moloco/docs/pull/2#discussion_r279588506
https://github.com/moloco/docs/pull/2#discussion_r279588981

File: android-sdk/README.md (hunk 8 above)
Comment (linjus) [reply android-sdk/README.md:3]: Yes, I think they agree with me.

Comment (soohan-ahn) [reply android-sdk/README.md:3]: ~~I am not sure I understood your comment properly, so tell me if there are any difference with your idea.~~
~~I understood as below.~~
~~* Keep title as `Moloco Android SDK`~~
~~* Use `Moloco SDK` on the description on here and below.~~

I have checked it by the direct message, and it was the issue of an article.

Comment (soohan-ahn) [reply android-sdk/README.md:3]: I have asked it to a native speaker, and got a response a naked noun is more natural.
Therefore, I will keep this as origin without modification this time.
Thank you for your comment!

**Comment by hyeonseo** [comment]
Looks good, ship it!

**Pull request merged**